hnn_core/cells_default.py

```diff
@@ -182,10 +187,7 @@ def basket(pos, cell_name='L2Basket', gid=None):
     for sec_name in p_secs:
         p_secs[sec_name]['sec_pts'] = sec_pts[sec_name]
 
-    cell = Cell(cell_name, pos=pos, gid=gid)
-    cell.build(p_secs, p_syn, topology, sect_loc)
-
-    return cell
+    return Cell(cell_name, pos, p_secs, p_syn, topology, sect_loc, gid)
```

Comment:
In general, it's good practice to use keyword arguments when calling/instantiating so it's future proof if certain arguments are removed added.
```suggestion
    return Cell(cell_name, pos, 
                      p_secs=p_secs, 
                      p_syn=p_syn, 
                      topology=topology, 
                      sect_loc=sect_loc, 
                      gid=gid)
```

Reply:
On a related note, it's kind of weird that `cell_name` and `pos` get flipped in the argument list for `pyramidal` and `basket` functions.

Reply:
is there a reason why we don't use keyword args for `cell_name` and `pos`?